iron-doc-behavior.html

```diff
@@ -111,6 +111,25 @@
         return prefix + type + suffix;
       },
 
+      /**
+       * Scrolls to the currently selected anchor, as identified
+       * by the URL hash. Whichever element or script is in charge
+       * of routing should call this method on initial page load and
+       * on hashchange events.
+       */
+      scrollToAnchor: function(hash) {
+        // ToDo: handle linking to private members
+        if (hash && hash.length > 1) {
+          // ensure all dom-repeats have rendered.
+          Polymer.dom.flush();
+          var anchorId = window.location.hash.slice(1);
```

Comment:
This uses `window.location.hash` but hash is passed in too? Probably want to just use the `hash` param?